src/main/java/com/android/volley/cronet/CronetHttpStack.java

```diff
@@ -244,27 +272,23 @@ private void setHttpMethod(Request<?> request, UrlRequest.Builder builder)
      *
      * @param request to get the headers from
      * @param additionalHeaders for the UrlRequest
-     * @param builder that we are adding the request headers to
+     * @param requestParameters parameters that we are adding the request headers to
      * @throws AuthFailureError is thrown if Request#getHeaders throws ones
      */
     private void setRequestHeaders(
-            Request<?> request, Map<String, String> additionalHeaders, UrlRequest.Builder builder)
+            Request<?> request,
+            Map<String, String> additionalHeaders,
+            CurlLoggedRequestParameters requestParameters)
```

Comment:
It's really a nitpick, but since we're adding to the requestParameters, I would suggest moving it as the first method parameter (same for other similar methods)

Reply:
Done